emmet-core/emmet/core/molecules/summary.py

```diff
@@ -0,0 +1,379 @@
+                    "open-shell molecule"
+    )
+    beta_population: List[NaturalPopulation] = Field(
+        None,
+        description="Natural electron populations of the beta electrons of an "
+                    "open-shell molecule"
+    )
+    alpha_lone_pairs: List[LonePair] = Field(
+        None,
+        description="Alpha electron lone pair orbitals of an open-shell molecule"
+    )
+    beta_lone_pairs: List[LonePair] = Field(
+        None,
+        description="Beta electron lone pair orbitals of an open-shell molecule"
+    )
+    alpha_bonds: List[Bond] = Field(
+        None,
+        description="Alpha electron bond-like orbitals of an open-shell molecule"
+    )
+    beta_bonds: List[Bond] = Field(
+        None,
+        description="Beta electron bond-like orbitals of an open-shell molecule"
+    )
+    alpha_interactions: List[Interaction] = Field(
+        None,
+        description="Alpha electron orbital-orbital interactions of an open-shell molecule"
+    )
+    beta_interactions: List[Interaction] = Field(
+        None,
+        description="Beta electron orbital-orbital interactions of an open-shell molecule"
+    )
+
+    # partial charges
+    partial_charges: Dict[str, List[float]] = Field(
+        None,
+        description="Atomic partial charges for the molecule using different partitioning schemes "
+                    "(Mulliken, Restrained Electrostatic Potential, Natural Bonding Orbitals, etc.)"
+    )
+
+    # partial spins
+    partial_spins: Dict[str, List[float]] = Field(
+        None,
+        description="Atomic partial spins for the molecule using different partitioning schemes "
+                    "(Mulliken, Natural Bonding Orbitals, etc.)"
+    )
+
+    # bonding
+    molecule_graphs: Dict[str, MoleculeGraph] = Field(
+        None,
+        description="Molecular graph representations of the molecule using different "
+                    "definitions of bonding."
+    )
+
+    bond_types: Dict[str, Dict[str, List[float]]] = Field(
+        description="Dictionaries of bond types to their length under different "
+                    "definitions of bonding, e.g. C-O to a list of the lengths of "
+                    "C-O bonds in Angstrom."
+    )
+
+    bonds: Dict[str, List[Tuple[int, int]]] = Field(
+        description="List of bonds under different definitions of bonding. Each bond takes "
+                    "the form (a, b), where a and b are 0-indexed atom indices",
+    )
+
+    bonds_nometal: Dict[str, List[Tuple[int, int]]] = Field(
+        description="List of bonds under different definitions of bonding with all metal ions "
+                    "removed. Each bond takes the form in the form (a, b), where a and b are "
+                    "0-indexed atom indices.",
+    )
+
+    # redox properties
+    electron_affinity: float = Field(None, description="Vertical electron affinity in eV")
+
+    ea_id: MPID = Field(None, description="MPID for electron affinity")
```

Comment:
The data type is called MPID. Unless we want to change that, it is strictly correct to say this is an MPID.

Reply:
Good point; I just wonder if that's going to lead to a messaging problem down the road because people may strongly associated MPID with solid materials. Might be a non-issue but just something that immediately occurred to me when I read this docstring. But you are right (in your other comment) that since the datatype is MPID the docstring is strictly correct.

Any thoughts @mkhorton ?

Reply:
I strongly concur with @rkingsbury's original comment: MP-ID (material ID) has to be exclusively for solid materials.

If we need to change/extend the MPID class to be extensible to additional identifiers (additional pre-fixes) and/or change the name of that class though, then maybe we can do that?

Reply:
As far as I can tell, that's already in place. The MPID data type can take a string or an int as input. In my builders, you can provide a prefix (for instance, the molecules in LIBE will be given the "libe" prefix).

For now, I'll change the wording in the docstring.